### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by cptpiepmatz**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
         "extract-i18n": {
           "builder": "@angular-devkit/build-angular:extract-i18n",
           "options": {
-            "browserTarget": "core:build"
+            "buildTarget": "core:build"
```
Comment (dr4hcu5-jan) [inline angular.json:108]: what is the difference between those targets?

**Review by dr4hcu5-jan — APPROVED** [review]

File: angular.json (hunk 1 above)
Comment (dr4hcu5-jan) [reply angular.json:108]: else lgtm

Comment (cptpiepmatz) [reply angular.json:108]: dunno, I ran `ng update` on the workspace and it did this

Comment (dr4hcu5-jan) [reply angular.json:108]: Well then, LGTM

**Pull request merged**